janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/computer/VertexMapJob.java

```diff
@@ -82,11 +82,11 @@ private VertexMapJob(IDManager idManager, FulgoraVertexMemory vertexMemory,
 
     @Override
     public VertexMapJob clone() {
-        ImmutableMap.Builder<MapReduce, FulgoraMapEmitter> cloneMap = ImmutableMap.builder();
+        Map<MapReduce, FulgoraMapEmitter> cloneMap = new HashMap<>(mapJobs.size());
         for (Map.Entry<MapReduce, FulgoraMapEmitter> entry : mapJobs.entrySet()) {
             cloneMap.put(entry.getKey().clone(), entry.getValue());
         }
-        return new VertexMapJob(idManager, vertexMemory, cloneMap.build());
+        return new VertexMapJob(idManager, vertexMemory, cloneMap);
     }
```

Comment:
No need making this map immutable because it is passed into the `VertexMapJob` and isn't used anywhere except `VertexMapJob` itself. Moreover the constructor  allows mutable maps to be passed (which is passed in `FulgoraGraphComputer`).